src/cljs/main/broadfcui/common/components.cljs

```diff
@@ -236,7 +236,7 @@
      {"methodVersion" (int (common/get-text refs "snapshotId"))})
    :render
    (fn [{:keys [props refs state this]}]
-     [:div {} (if (:parse-error? props) (style/create-server-error-message "ERROR: This method does not pass WDL validation."))
+     [:div {} (if-let [wdl-parse-error (:wdl-parse-error props)] (style/create-server-error-message wdl-parse-error))
```

Comment:
OK, changing this to a `when-let`.

The `some->` notation is interesting but I don't like it yet :)